pkgs/by-name/sh/shdotenv/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gnuawk
+,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "shdotenv";
+  version = "0.14.0";
+
+  src = fetchFromGitHub {
+    owner = "ko1nksm";
+    repo = "shdotenv";
+    rev = "v${version}";
+    hash = "sha256-3O9TUA3/vuC12OJTxVVoAGmgSRq+1xPG7LxX+aXqVCo=";
+  };
+
+  buildInputs = [
+    gnuawk
+  ];
```

```suggestion
, gawk
,
}:

stdenv.mkDerivation rec {
  pname = "shdotenv";
  version = "0.14.0";

  src = fetchFromGitHub {
    owner = "ko1nksm";
    repo = "shdotenv";
    rev = "v${version}";
    hash = "sha256-3O9TUA3/vuC12OJTxVVoAGmgSRq+1xPG7LxX+aXqVCo=";
  };

  buildInputs = [
    gawk
  ];
```
the gnu-awk package is named `gawk`